repository_nuget_test.go

```diff
@@ -42,6 +42,7 @@ func getTestRepositoryNugetProxy(name string) Repository {
 		RepositoryNegativeCache: &RepositoryNegativeCache{},
 		RepositoryNugetProxy: &RepositoryNugetProxy{
 			QueryCacheItemMaxAge: 1,
+			NugetVersion:         "V3",
```

Comment:
why not referencing the constant? `NugetVersion3`

Reply:
i want to test the value that we get from nexus provider